docs/book.toml

```
@@ -1,14 +1,12 @@
 [book]
-title = "The Miden protocol"
-authors = ["Miden team"]
-description = "Description and core structures for the Miden Rollup protocol"
+authors      = ["Miden team"]
+description  = "Description and core structures for the Miden rollup protocol"
+language     = "en"
 multilingual = false
-language = "en"
+title        = "The Miden Node"
```

```suggestion
title        = "The Miden Protocol"
```

per https://github.com/0xPolygonMiden/miden-base/pull/1190#discussion_r1995025997